src/helpers/types.ts

```diff
@@ -42,7 +42,13 @@ export function wrapWithTypeOptions<T extends GraphQLType>(
 ): T {
   let gqlType: GraphQLType = type;
   if (typeOptions.array) {
-    gqlType = new GraphQLList(new GraphQLNonNull(gqlType));
+    if (typeOptions.nullable === "item") {
```

Comment:
A bit too complicated - please change this function body to:
```ts
  let gqlType: GraphQLType = type;
  if (typeOptions.array) {
    if (typeOptions.nullable === "item" || typeOptions.nullable === "itemAndList") {
      gqlType = new GraphQLList(gqlType);
    } else {
      gqlType = new GraphQLList(new GraphQLNonNull(gqlType));
    }
  }
  if (!typeOptions.nullable || typeOptions.nullable === "item") {
    gqlType = new GraphQLNonNull(gqlType);
  }
  return gqlType as T;
```

Reply:
I don't think this will make it simpler since there may have some weird behavior when a non-array field annotated as `@Field( nullalbe = 'items' )`. it would be better just limit those options only for array.

Reply:
We should just throw an error to the user when a non-array field is annotated as `nullable: "items"`. Silently converting `items` or `itemsAndList` to `true` (which is happening right now) is not a good idea.